packages/trader/src/Modules/Reports/Containers/statement.jsx

```diff
@@ -122,12 +167,15 @@ const Statement = ({
     if (error) return <p>{error}</p>;
 
     const filter_component = (
-        <CompositeCalendar
-            input_date_range={filtered_date_range}
-            onChange={handleDateChange}
-            from={date_from}
-            to={date_to}
-        />
+        <>
+            <CompositeCalendar
+                input_date_range={filtered_date_range}
+                onChange={handleDateChange}
+                from={date_from}
+                to={date_to}
+            />
+            <TransactionFilter handleFilterChange={handleFilterChange} />
```

Comment:
We show account statistics only for `iom` clients. Currently, account statistics UI breaks after adding this dropdown. Can you check this for `iom` account? 🙏

Reply:
The design team already update the styling and I've implemented the changes, please help re-review